plugins/shared-react/package.json

```diff
@@ -24,6 +24,7 @@
   },
   "dependencies": {
     "@kubernetes/client-node": "^0.18.1",
+    "@patternfly/react-icons": "^4.93.6",
```

Comment:
We should avoid creating a dependency on patternfly in this package

Reply:
In that case we might want to keep icons in the plugins itself. Wdyt?  @debsmita1

Reply:
can we use icons from MUI? over pf/react-icons? 

if yes we can keep it here else it's not ideal to bring in pf dependency in a shared package unless it's really needed

Reply:
I will add the similar icons from MUI then.